### PR TITLE
Add a launcher screen. bounty #140

### PR DESCRIPTION
**Pull request by darkracer**
The screen skips after 1 second.

### CODE REVIEW

**Comment by adrianonco** [comment]
@darkracer, I've commented in https://assembly.com/giraff/bounties/140#

**Comment by alouanemed** [comment]
@darkracer  Can you add a screenshot ?

**Comment by alouanemed** [comment]
@darkracer it's all done ?

**Comment by darkracer** [comment]
I used my own cropped assets and I think it's done.

**Comment by alouanemed** [comment]
@darkracer  Can you update the asset to one @adrianonco  sent you ?

**Comment by alouanemed** [comment]
@darkracer  Also , the naming convention of Launcher_Android.png is incorrect !
It must contain only lowercase letters and digits ([a-z0-9_.])

File: app/src/main/java/assembly/giraff/LauncherActivity.java [hunk 1]
```diff
@@ -16,6 +21,18 @@
...
+            @Override
+            public void run() {
+                Button facebook = (Button)findViewById(R.id.facebookButton);
+                facebook.setVisibility(View.VISIBLE);
+                Button mainActivity = (Button)findViewById(R.id.mainButton);
+                mainActivity.setVisibility(View.VISIBLE);
+                RelativeLayout layout = (RelativeLayout)findViewById(R.id.launcherLayout);
+                layout.setBackground(null);
+            }
+        }, 1000);
```
Comment (alouanemed) [inline app/src/main/java/assembly/giraff/LauncherActivity.java]: Change this to 4000 or more

**Comment by alouanemed** [comment]
I am going to merge this ,and change what should be changed :)

**Pull request merged**